content/post/2022-08-12-friday-hacks-223.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Friday Hacks #223, August 12: Welcome Tea"
+date: 2022-08-03 22:34:26.523646
+author: Shen Yi Hong
+url: /2022/08/friday-hacks-223
+aliases:
+  - /welcometea2022
+nofh: true
+---
+
+<em>In the jargon of the computer programmer, a hacker is someone who strives to solve problems in elegant and ingenious ways.</em>
+
+**Date/Time:** Friday, August 12 at 7:00pm<br />
+**Venue:** TBC<br />
+
+## Welcome Tea
+
+#### Talk Description:
+
+Welcome back to Friday Hacks 👋
+
+We will be sharing about what we do and the importance of fostering hacker culture in NUS. We will also give an overview of the events that we have lined up for the upcoming academic year and how you can get involved in helping us spread the joy of hacking!
+
+All freshmen and current students from all faculties, as well as anyone else who is interested, are all welcome to attend!
+
+If you are unable to come join us physically, you are welcome to join us [online](https://nus-sg.zoom.us/j/89065336025?pwd=a0d4ZkRQMHZwNUFEL2hMcFFwZUNVdz09)!
```

Comment:
Is this link confirmed?

Reply:
i think we can just use this? its hosted on my account

Reply:
Ok!